### DISCUSSION TITLE
Bookmarks bar in new tab

### DISCUSSION THREAD

**Discussion by davidk-caditglobal** [dailydotdev/daily#1348] [Feature Requests]
Is it possible to add a bookmarks bar in a new tab? 
![image](https://github.com/user-attachments/assets/01716763-8dc5-4e9a-a7b2-a46c680971ac)

Because Chrome by default shows a bookmark bar on new tabs if the "Bookmarks Bar" settings is turned off 
![image](https://github.com/user-attachments/assets/643fc264-6669-48c6-9ddb-e8893d2509b7)

https://github.com/dailydotdev/daily/discussions/1047

**Comment by nimrodkra** [comment]
Unfortunately it's not possible since Chrome itself doesn't allow us to
control this aspect of the browser. So the options are either you turn the
bookmarks bar on (and it will be always on - in the new tab and afterwards
when you navigate) or to turn it off (the current situation you're in).
Honestly, I wish we could do more to solve it but it's technically not
possible.

**Comment by davidk-caditglobal** [comment]
I see, thank you for the explanation!!
